l10n_it_fatturapa_pec/models/mail_thread.py

```diff
@@ -180,6 +180,19 @@ def find_attachment_by_subject(self, subject):
     def create_fatturapa_attachment_in(self, attachment):
         decoded = base64.b64decode(attachment.datas)
         fatturapa_attachment_in = self.env['fatturapa.attachment.in']
+        fetchmail_server_id = self.env.context.get('fetchmail_server_id')
+        company_id = False
+        e_invoice_user_id = False
+        if fetchmail_server_id:
+            sdi_chan = self.env['sdi.channel'].search([
+                ('fetch_pec_server_id', '=', fetchmail_server_id)], limit=1)
+            if sdi_chan:
+                # See check_fetch_pec_server_id
```

Comment:
Ho guardato `check_fetch_pec_server_id` ma non ho colto perché l'ho dovuto fare, potresti aggiungere una spiegazione?

Reply:
@SimoRubi Ok! La logica è che facendo un mail fetch, ci ritroviamo una fattura elettronica in arrivo a cui dobbiamo associare una company, l'unico modo che abbiamo è quello di andare a cercare qual'è l'SDI channel utilizzato, tramite ``fetch_pec_server_id``, e da questo ricaviamo la company da utilizzare per registrare la fattura in questione.